pkg/config/template.go

```diff
@@ -735,6 +740,11 @@ irqbalance_config_file = "{{ .IrqBalanceConfigFile }}"
 
 `
 
+const templateStringCrioRuntimeRdtConfigFile = `# Path to the RDT configuration file for configuring the resctrl pseudo-filesystem.
+rdt_config_file = "{{ .RdtConfigFile }}"
+
```

Comment:
Let's mention that this support configuration reload, too.

Reply:
Addressed